### ISSUE TITLE
Allow easy switching between Remote-WSL and normal VSCode

### ISSUE THREAD

**Issue by labooner** [microsoft/vscode-remote-release#510]
`Remote - WSL` only allows us to open a "New Window". Remote - Containers offers:

- Reopen folder in Container
- Open folder in Container

![image](https://user-images.githubusercontent.com/4803529/58587897-5d8f9500-8256-11e9-90e1-f9f906f77421.png)
> Clicking the "Remotes" button in the bottom left with `Remotes - WSL` and `Remotes - Containers` plugins installed.

In the short time I've been using the "Remote" functionality, I've frequently found I open VS Code, open a "Remote-WSL: New Window" and then close the original. I then have to open my folder again.

Additionally, once I'm in a Remote-WSL window, I can no longer go back to a "vanilla" window. If I'm in a Remote-Containers window, I can do `Reopen Folder Locally`.

----

To summarise, I'd like the following:

- `Reopen Folder in WSL`
- `Reopen Folder Locally` 

Thanks for this feature. Its been a long time since I've been this excited about some IDE functionality!

**Comment by kieferrm** [comment]
@labooner for clarification: You want to be able to easily toggle between a local and a WSL window, both of which are open on the same folder?

**Comment by labooner** [comment]
Correct. Thanks!

**Comment by aeschli** [comment]
Test plan item: https://github.com/microsoft/vscode-remote-release/issues/790
